contracts/access/IAccessControlDefaultAdminRules.sol

```diff
@@ -12,62 +12,137 @@ import "./IAccessControl.sol";
      */
-    function defaultAdminTransferDelayedUntil() external view returns (uint48);
+    function defaultAdminDelayIncreaseWait() external view returns (uint48);
 
     /**
-     * @dev Starts a `DEFAULT_ADMIN_ROLE` transfer by setting a pending default admin
-     * and a timer to pass.
+     * @dev Returns the delay between each {defaultAdmin} transfer.
      *
-     * Requirements:
+     * A scheduled delay change will take effect as soon as the schedule passes, returning the new delay.
+     */
+    function defaultAdminDelay() external view returns (uint48);
+
+    /**
+     * @dev Returns a tuple of `newDelay` and `schedule`.
+     * The former will be the new {defaultAdminDelay} after the latter passes.
      *
-     * - Only can be called by the current `DEFAULT_ADMIN_ROLE` holder.
+     * A zero value in both indicates there's no pending delay change.
      *
-     * Emits a {DefaultAdminRoleChangeStarted}.
+     * NOTE: A zero value only for `newDelay` means that the next {defaultAdminDelay} will be zero.
      */
-    function beginDefaultAdminTransfer(address newAdmin) external;
+    function pendingDefaultAdminDelay() external view returns (uint48 newDelay, uint48 schedule);
 
     /**
-     * @dev Completes a `DEFAULT_ADMIN_ROLE` transfer.
+     * @dev Starts a {defaultAdmin} transfer by setting {pendingDefaultAdmin} with a {defaultAdminDelay} schedule.
      *
      * Requirements:
      *
-     * - Caller should be the pending default admin.
+     * - Only can be called by the current {defaultAdmin}.
+     *
+     * Emits a DefaultAdminRoleChangeStarted event.
+     */
+    function beginDefaultAdminTransfer(address newAdmin) external;
+
+    /**
+     * @dev Completes a {defaultAdmin} transfer.
+     *
      * - `DEFAULT_ADMIN_ROLE` should be granted to the caller.
      * - `DEFAULT_ADMIN_ROLE` should be revoked from the previous holder.
+     *
+     * Requirements:
+     *
+     * - Only can be called by the {pendingDefaultAdmin}'s `newAdmin` and if its `scheduled` has passed.
      */
     function acceptDefaultAdminTransfer() external;
 
     /**
-     * @dev Cancels a `DEFAULT_ADMIN_ROLE` transfer.
+     * @dev Cancels a {defaultAdmin} transfer if there was a {pendingDefaultAdmin}.
+     *
+     * A {pendingDefaultAdmin} not yet accepted can also be cancelled with this function.
      *
      * Requirements:
      *
-     * - Can be called even after the timer has passed.
-     * - Can only be called by the current `DEFAULT_ADMIN_ROLE` holder.
+     * - Only can be called by the current {defaultAdmin}.
+     *
+     * May emit a DefaultAdminTransferCanceled event.
      */
     function cancelDefaultAdminTransfer() external;
+
+    /**
+     * @dev Begins a {defaultAdminDelay} change by scheduling the change in a way in which the
+     * current delay is still guaranteed to be respected.
+     *
+     * The {pendingDefaultAdminDelay}'s schedule is defined such that `(its schedule + a delayed default admin transfer)`
```

Comment:
> its schedule + a delayed default admin transfer

That is not very clear

Reply:
I improved the way is described. I'm solving since I think it should be clear now